diff --git a/lib/index.d.ts b/lib/index.d.ts
index 9823e7cc4df0294c575eec618d903beb80142287..863941dd77a53be4d61928b6d10e3787bac78620 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,5 +1,5 @@
-import { createNextImportResolver, resolve } from "./resolve";
-import type { NewResolver, Options, PackageGlobOptions, PackageOptions } from "./types";
+import { createNextImportResolver, resolve } from "./resolve.js";
+import type { NewResolver, Options, PackageGlobOptions, PackageOptions } from "./types.js";
 declare const interfaceVersion = 2;
 export type { NewResolver, Options, PackageGlobOptions, PackageOptions };
 export { createNextImportResolver, interfaceVersion, resolve };
diff --git a/lib/resolve.d.ts b/lib/resolve.d.ts
index e2f2c3da60376863fe3cbdf3bb1bdc465bc7c1b4..78dd32bd3e5c2ba7f5ebae827fb85fc5debb35b8 100644
--- a/lib/resolve.d.ts
+++ b/lib/resolve.d.ts
@@ -1,3 +1,3 @@
-import type { NewResolver, Options, ResolvedResult } from "./types";
+import type { NewResolver, Options, ResolvedResult } from "./types.js";
 export declare function resolve(modulePath: string, sourceFile: string, config?: Options | null): ResolvedResult;
 export declare function createNextImportResolver(config?: Options | null): NewResolver;
